samples/RT-sample-app/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,149 @@
+  "developer": {
+    "name": "Teams App, Inc.",
+    "websiteUrl": "https://www.example.com",
+    "privacyUrl": "https://www.example.com/privacy",
+    "termsOfUseUrl": "https://www.example.com/termofuse"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "RT-Sample-App",
+    "full": "full name for RT-sample-app"
+  },
+  "description": {
+    "short": "Specify runtime requirements for specific app capabilities",
+    "full": "Specify runtime requirements for specific app capabilities"
+  },
+  "accentColor": "#FFFFFF",
+  "elementRelationshipSet": {
+    "oneWayDependencies": [
+      {
+        "element": {
+          "name": "composeExtensions",
+          "id": "composeExt_ID",
+          "commandIds": [
+            "createCard"
+          ]
+        },
+        "dependsOn": [
+          {
+            "name": "bots",
+            "id": "${{BOT_ID}}"
+          }
+        ]
+      }
+    ],
+    "mutualDependencies": [
+      [
+        {
+          "name": "staticTabs",
+          "id": "staticTab_ID"
+        },
+        {
+          "name": "composeExtensions",
+          "id": "composeExt_ID"
+        },
+        {
+          "name": "bots",
+          "id": "${{BOT_ID}}"
+        }
+      ]
+    ]
+  },
+  "elementRelationshipSet": {
+    "oneWayDependencies": [
+      {
+        "element": {
+          "name": "staticTabs",
+          "id": "staticTab_Id"
+        },
+        "dependsOn": [
+          {
+            "name": "composeExtensions",
+            "id": "composeExt_ID",
+            "commandIds": [
+              "createCard"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  "bots": [
+    {
+      "botId": "${{BOT_ID}}",
+      "needsChannelSelector": false,
+      "isNotificationOnly": false,
+      "scopes": [
+        "personal",
+        "groupchat",
+        "team"
+      ]
+    }
+  ],
+  "composeExtensions": [
+    {
+      "id": "composeExt_ID",
+      "botId": "${{BOT_ID}}",
+      "commands": [
+        {
+          "id": "createCard",
+          "context": [
+            "compose",
+            "message",
+            "commandBox"
+          ],
+          "description": "Command to run action to create a Card from Compose Box",
+          "title": "Create Card",
+          "type": "action",
+          "parameters": [
+            {
+              "name": "title",
+              "title": "Card title",
+              "description": "Title for the card",
+              "inputType": "text"
+            },
+            {
+              "name": "subTitle",
+              "title": "Subtitle",
+              "description": "Subtitle for the card",
+              "inputType": "text"
+            },
+            {
+              "name": "text",
+              "title": "Text",
+              "description": "Text for the card",
+              "inputType": "textarea"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "staticTabs": [
```

Comment:
There should also be "hostMustSupport dialogUrl" defined under staticTabs.

Reply:
Sample updated with **hostMustSupport dialogUrl**